packages/kbn-ui-framework/dist/kui_dark.css

```diff
@@ -1,10 +1,24 @@
+/* 1 */
```

Comment:
is this double `/* 1 */` comment intentional, @cchaos ?

Reply:
No, but this is the compiled file for an old system I'm not too worried about comments.